app/api/revalidate/route.ts

```diff
@@ -0,0 +1,17 @@
+import { revalidatePath } from 'next/cache';
```

Comment:
@terotik @lilia1891 This can be used to invalidate the Next.js cache (which we're not fully using _yet_). GET `<url>/api/revalidate?path=/` would invalidate all cache for all routes.

Reply:
:white_check_mark: